src/store/index.js

```diff
@@ -2,9 +2,11 @@ import Vue from 'vue';
 import Vuex from 'vuex';
 
 import mutations from './mutations';
+// eslint-disable-next-line import/no-cycle
```

Comment:
I propose to leave it for future refactoring. When components will depend on Backend module of the store instead of backend methods directly then we will be able to get rid of cycle dependencies.

Components shouldn't depend on the backend methods file because that file obviously depends on the store (to track backend status), but the components shouldn't depend on the store explicitly for testing/flexibility purposes.